packages/catalog-search/src/styles/_CurrentRefinements.scss

```diff
@@ -1,5 +1,6 @@
 .fe__refinement-badge {
   border-radius: 0;
+  cursor: pointer;
```

Comment:
Including this small change to make refinement badges more responsive by changing the cursor to show they are clickable.